_pages/en/it-picture-long-term.md

```diff
@@ -83,11 +91,16 @@ Ecosystem of services are created by leveraging private sector expertise where o
 
 #### Open Collaboration
 
-- ESDC custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back
+- ESDC custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back ( e.g. The Canadian version of [code.gov](https://code.gov/))
 - ESDC data (performance and automated decision making statistics) published as open datasets
 
 ### IITB-ESDC
 
+- Technology is used in all of ESDC's service delivery
+- IT services are able to response in a timely manner to the ever growing pace of business change
+- The relationship between ESDC Service lines and IITB is harmonized where business understands the risk of IT and the success of IT is measured via business success
```

Comment:
Not tangible ... I propose 
"- IT success is measured through business success"

I was going to just change it, but I wasn't exactly sure that's what you wanted to say.

Reply:
Agree that #2 is a repeat of #3. Will remove it.
I think #1 (tech used in all services) is important to highlight. It's a statement that services cannot be delivered without tech.
#3 is a reference to War Peace IT. It's not just success, it's also risk.

Reply:
See commit #02a1519 with rewording